components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/internal/IdentityRecoveryServiceDataHolder.java

```diff
@@ -344,4 +347,24 @@ public void setApplicationManagementService(ApplicationManagementService applica
 
         this.applicationManagementService = applicationManagementService;
     }
+
+    /**
+     * Set WorkflowManagementService OSGi service.
+     *
+     * @param workflowManagementService Workflow Management Service.
+     */
+    public void setWorkflowManagementService(WorkflowManagementService workflowManagementService) {
+
+        this.workflowService = workflowManagementService;
+    }
+
+    /**
+     * Get WorkflowManagementService OSGi service.
+     *
+     * @return Workflow Management Service.
+     */
+    public WorkflowManagementService getWorkflowManagementService() {
```

Comment:
This change provides `getWorkflowManagementService` in `IdentityRecoveryServiceDataHolder`, which is good because it allows other services to access the `WorkflowManagementService` through a proper accessor method, improving encapsulation and reusability.